conf/messages

```diff
@@ -438,9 +439,12 @@ ileQueryResponse.timeout.message = Try again later.
 sessionTimout.title = For your security, we signed you out
 sessionTimout.paragraph.saved = We saved your answers.
 sessionTimout.signin.button = Sign in
-sessionTimout.back.button = Back to GOV.UK
 
 timeoutDialog.title = Your session will timeout soon
 timeoutDialog.message = For your security, we will sign you out in
 timeoutDialog.keepAlive.text= Stay signed in
-timeoutDialog.signout.text= Sign out 
\ No newline at end of file
+timeoutDialog.signout.text= Sign out
+
+userSignedOut.title = We have saved your answers and signed you out
+userSignedOut.information = To continue the declaration later, save this link:
+userSignedOut.startPageLink = https://www.tax.service.gov.uk/customs-movements/start
```

Comment:
You need to remove the production domain @MaciejRewera

Reply:
Would it not be better to use the url defined in application.conf?
`urls.loginContinue`

Reply:
```
userSignedOut.startPageLink = {0}/customs-declare-exports/start
```
then pass the `@request.host` form the view.
This is a solution.

Reply:
👍